src/patternfly/layouts/Grid/docs/code.md

```diff
@@ -1,14 +1,14 @@
 ## Overview
 
-A Grid layout framework including responsive columns, row spans and gutters 
+A Grid layout framework including responsive columns, row spans and gutters. **Breakpoint is optional. Breakpoints: -on-xs, -on-sm, -on-md, -on-lg, -on-xl**
```

Comment:
Should these classes be in the usage table as well?

Reply:
Updated the usage doc to better show how to form responsive classes